go.mod

```diff
@@ -93,6 +93,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.3.0 // indirect
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
 	github.com/cilium/ebpf v0.16.0 // indirect
+	github.com/containerd/containerd v1.7.23 // indirect
```

Comment:
Is introducing a dependency on a previous version of containerd going to cause problems?

Reply:
Yeah, this was because it was still vendoring old tag of hcsshim. It should be fixed now.